local/deployment/full-irs/subcharts/irs-frontend-service/values.yaml

```
@@ -1,6 +1,7 @@
 ---
 replicaCount: 1
 
+# Which image should be used here?
 image:
   repository: ghcr.io/catenax-ng/product-item-relationship-service-frontend
```

This repo was not moved to tractusx. Repository stays at it is.
```suggestion
image:
  repository: ghcr.io/catenax-ng/product-item-relationship-service-frontend
```